bullet-train-core.js

```diff
@@ -59,7 +59,6 @@ const BulletTrain = class {
             traits.forEach(trait => {
                 userTraits[trait.trait_key.toLowerCase().replace(/ /g, '_')] = trait.trait_value
             });
-            this.oldFlags = flags;
```

Comment:
This line and the next, in this order, are not helping to detect differences between them in the onChange callback, at least directly, without using another variable.